### PR TITLE
Added tab chooser large and updated chooser css with the chooser css …

### PR DESCRIPTION
**Pull request by Pallavi-PH**
…present in maya-ui

### CODE REVIEW

**Comment by Pallavi-PH** [comment]
Screenshot from 2020-03-09 17-14-06

File: scss/product/components/_chooser.scss [hunk 1]
```diff
@@ -1,58 +1,95 @@
...
+      padding: 8px 6px;
+      span{
+        height: 2.5rem;
+        width: 2.5rem;
+        margin:15px 0;
+      }
+      img {
+        height: 2.5rem;
+        width: 2.5rem;
+      }
```
Comment (visualapps) [inline scss/product/components/_chooser.scss]: Do not style the element directly use the class instead, it was` tab-chooser-logo` for img tag

Comment (Pallavi-PH) [reply scss/product/components/_chooser.scss]: Done

File: scss/product/components/_chooser.scss [hunk 2]
```diff
@@ -1,58 +1,94 @@
...
+      .chooser-card-body{
+        border-top: $chooser-card-item-body-border-color-active;
+
+        .chooser-card-label {  
+          color: $chooser-card-item-label-color-active;  
+        }
+      }
+  
+    }
+    &.chooser-card-col-5 {
```
Comment (visualapps) [inline scss/product/components/_chooser.scss]: where we using this class? can you point me to html?

Comment (Pallavi-PH) [reply scss/product/components/_chooser.scss]: It is used in `nav-tab-avatar` component

Comment (Pallavi-PH) [reply scss/product/components/_chooser.scss]: I have replaced `chooser-col-5` with maya-ui

File: scss/core/utilities/_sizing.scss [hunk 3]
```diff
@@ -11,6 +11,34 @@
 .mw-100 { max-width: 100% !important; }
 .mh-100 { max-height: 100% !important; }
 
+//Widths
+
+$widths: (
```
Comment (visualapps) [inline scss/core/utilities/_sizing.scss]: should this go inside variable file?

File: scss/core/utilities/_sizing.scss [hunk 4]
```diff
@@ -11,6 +11,34 @@
...
+  "6xl" :  12,
+  "7xl" :  16,
+  "8xl" :  24,
+  "9xl" :  32,
+  "10xl":  40,
+  "11xl":  48,
+);
+
+@each $size, $value in $widths {
+  @debug 1rem * $value;
```
Comment (visualapps) [inline scss/core/utilities/_sizing.scss]: need to remove after testing

File: scss/core/utilities/_sizing.scss (hunk 3 above)
Comment (Pallavi-PH) [reply scss/core/utilities/_sizing.scss:16]: Raised separate PR for this

File: scss/core/utilities/_sizing.scss (hunk 4 above)
Comment (Pallavi-PH) [reply scss/core/utilities/_sizing.scss:36]: Raised separate PR for this

**Pull request merged**